diff --git a/node_modules/react-pts-canvas/dist/index.es.js b/node_modules/react-pts-canvas/dist/index.es.js
index 0b6af6b..4695a2f 100644
--- a/node_modules/react-pts-canvas/dist/index.es.js
+++ b/node_modules/react-pts-canvas/dist/index.es.js
@@ -140,8 +140,8 @@ var PtsCanvas = function (_React$Component) {
 
       return React.createElement(
         'div',
-        { className: this.props.name || "", style: this.props.style },
-        React.createElement('canvas', { className: this.props.name ? this.props.name + "-canvas" : "", ref: function ref(c) {
+          {className: this.props.className || null},
+        React.createElement('canvas', { ref: function ref(c) {
             return _this2.canvRef = c;
           }, style: this.props.canvasStyle })
       );
diff --git a/node_modules/react-pts-canvas/dist/index.js b/node_modules/react-pts-canvas/dist/index.js
index a5f6f90..aaff789 100644
--- a/node_modules/react-pts-canvas/dist/index.js
+++ b/node_modules/react-pts-canvas/dist/index.js
@@ -146,8 +146,8 @@ var PtsCanvas = function (_React$Component) {
 
       return React.createElement(
         'div',
-        { className: this.props.name || "", style: this.props.style },
-        React.createElement('canvas', { className: this.props.name ? this.props.name + "-canvas" : "", ref: function ref(c) {
+          {className: this.props.className || null},
+        React.createElement('canvas', { ref: function ref(c) {
             return _this2.canvRef = c;
           }, style: this.props.canvasStyle })
       );
@@ -157,7 +157,6 @@ var PtsCanvas = function (_React$Component) {
 }(React.Component);
 
 PtsCanvas.defaultProps = {
-  name: "pts-react", // maps to className of the container div
   background: "#9ab",
   resize: true,
   retina: true,
@@ -200,7 +199,6 @@ var QuickStartCanvas = function (_PtsCanvas) {
 }(PtsCanvas);
 
 QuickStartCanvas.defaultProps = {
-  name: "pts-react", // maps to className of the container div
   background: "#9ab",
   resize: true,
   retina: true,
